examples/tutorials/pong/steps/step6/main.py

```diff
@@ -0,0 +1,109 @@
+import kivy
+kivy.require('1.1.1')
+
+from kivy.app import App
+from kivy.uix.widget import Widget
+from kivy.properties import NumericProperty, ReferenceListProperty,\
```

Comment:
That `\` is already present in the code, i didnt touch that.

Look here https://github.com/kivy/kivy/blob/master/examples/tutorials/pong/main.py

Reply:
I thought so, yet if there is some old template, it doesn't really make sense to preserve old style if there is a way to make it prettier. You can leave it alone if you want consistency with the old file though.

Reply:
Fixing old ones seems better, because the step1 will be sawn first, then introducing bad things to newcomers. Sed power and/or replace in all documents is our friend.